homeassistant/components/wink.py

```diff
@@ -77,13 +78,25 @@ def setup(hass, config):
                                     client_secret)
 
     global SUBSCRIPTION_HANDLER
```

Comment:
We should use hass.data for this while we're refactoring, better not to pollute the global space where possible.